manifests/h/Hibit/StartUpManager/2.3.20.100/Hibit.StartUpManager.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Hibit.StartUpManager
+PackageVersion: 2.3.20.100
+PackageLocale: en-US
+Publisher: HiBitSoftware
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: HiBit Startup Manager version 2.3.20.100
```

```suggestion
PackageName: HiBit Startup Manager
```
Versions should be kept out of package names